lib/storage_tables/engine.rb

```diff
@@ -27,6 +27,12 @@ class Engine < ::Rails::Engine
       end
     end
 
+    initializer "storage_tables.configs" do
+      config.after_initialize do |app|
+        StorageTables.routes_prefix = app.config.storage_tables.routes_prefix || "/rails/storage_tables"
```

Comment:
Hebben we deze prefix ook aangepast in debiteurenbeheer? Ik zie soms "rails/active_storage" in download urls in de frontend. Vind ik niet echt netjes om op die manier te exposen hoe we onze documenten opslaan. Ik zou dan liever iets generieks als "assets" gebruiken.

Reply:
Er worden nog wel bestanden op die manier exposed. Daar zouden we even naar moeten kijken dan